api/server/server.go

```diff
@@ -541,8 +551,8 @@ type routesResponse struct {
 }
 
 type callResponse struct {
-	Message string       `json:"message"`
-	Call    *models.Call `json:"call"`
+	Message string           `json:"message"`
+	Call    *models.CallBase `json:"call"`
```

Comment:
yea i think just make that json field a `json:"-"` instead of having an extra model we have to dangle around

Reply:
fixed